modules/nixos/monitoring/grafana.nix

```diff
@@ -0,0 +1,69 @@
+{ config, ... }:
+{
+  systemd.services.grafana.after = [ "prometheus.service" ];
+
+  services.grafana = {
+    enable = true;
+    settings = {
+      analytics.reporting_enabled = false;
+      analytics.feedback_links_enabled = false;
+
+      "auth.anonymous".enabled = false;
```

Comment:
Don't see much point in making this public, only really of interest to nix-community org members.

Reply:
We will actually want this public for logs from nixpkgs-updates with loki/promtail so I'll just leave it enabled.